nbs/examples/MLFlow_and_neuralforecast.ipynb

```diff
@@ -0,0 +1,468 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #10.    model_name = "Neuralforecast_model"
</pre><p>please move this down to the register_model call</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Nixtla/neuralforecast/pull/892/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>

Reply:
Change added via commit 9a538ae3f7873b9c67ba6f333ac9100f82c214fc
Thank you all these suggestions.   :)